fathom/src/surface.rs

```diff
@@ -205,27 +204,26 @@ pub enum Term<'arena, Range> {
     /// Arrow types.
     Arrow(
         Range,
-        Plicity,
```

Comment:
Why remove implicit arrow types?

Reply:
Eventually I was hoping that we could make implicit arguments named. I’m also not sure how useful implicit arrows are? Seeing as you can’t infer them based on other arguments or the return type?

Reply:
Hmm, true. Since the most common use case for implicit parameters are type parameters, unnamed type parameters wouldn't be very useful.